lib/utils.js

```diff
@@ -19,6 +18,17 @@ function isArray(val) {
   return toString.call(val) === '[object Array]';
 }
 
+
+/**
+ * Determine if a value is a Buffer
+ *
+ * @param {Object} val The value to test
+ * @returns {boolean} True if value is a Buffer, otherwise false
+ */
+function isBuffer(val) {
+  return ![undefined, null].includes(val) && val.constructor === Buffer;
```

Comment:
consider `val != null` here if the goal is to be equivalent to `val !== null && val !== undefined`

Reply:
also consider checking `typeof Buffer !== "undefined"` so that we don't get strict mode errors related to the Buffer variable not being defined (browser tests should catch this?)

Reply:
just do this

```js
  return val != null && val.constructor != null &&
typeof val.constructor.isBuffer === 'function' && val.constructor.isBuffer(val)
```

Reply:
Why not simplify with `Buffer.isBuffer` ? 
If the package is installed via npm, we can assume it's running in Node and has `Buffer` defined.
Otherwise, it gets polyfilled during Webpack builds for browsers.

Reply:
It's even used in other places of the code, like here: https://github.com/axios/axios/blob/master/lib/adapters/http.js#L41

Reply:
Nevermind- I tried it and it injects the polyfill for the whole Buffer module. Your solution works best.

Reply:
So, are you going to merge it? Do you want me to do anything else?